### PR TITLE
8285097: Duplicate XML keys in XPATHErrorResources.java and XSLTErrorResources.java

### PR DESCRIPTION
**Pull request by shruacha1234**
Removing the Duplicate keys present in XSLTErrorResources.java and XPATHErrorResources.java

The bug report for the same: https://bugs.openjdk.java.net/browse/JDK-8285097

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Issue
 * [JDK-8285097](https://bugs.openjdk.java.net/browse/JDK-8285097): Duplicate XML keys in XPATHErrorResources.java and XSLTErrorResources.java

### Reviewers
 * [Joe Wang](https://openjdk.java.net/census#joehw) (@JoeWang-Java - **Reviewer**)
 * [Tyler Steele](https://openjdk.java.net/census#tsteele) (@backwaterred - Author) ⚠️ Review applies to [c294a150](https://git.openjdk.java.net/jdk/pull/8318/files/c294a1508815a42c11ff4bfd01568999d4787a0b)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8318/head:pull/8318` \
`$ git checkout pull/8318`

Update a local copy of the PR: \
`$ git checkout pull/8318` \
`$ git pull https://git.openjdk.java.net/jdk pull/8318/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8318`

View PR using the GUI difftool: \
`$ git pr show -t 8318`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8318.diff">https://git.openjdk.java.net/jdk/pull/8318.diff</a>

</details>

### CODE REVIEW

**Comment by backwaterred** [comment]
@shruacha1234, a few notes specific to openJDK PRs:
- Please change the issue name to: "8285097: Duplicate XML keys in XPATHErrorResources.java and XSLTErrorResources.java" this will correctly link the issue in bugs.openjdk.net
- Please enable github actions. This allows the pre-tests to run.

**Comment by shruacha1234** [comment]
This contribution is on behalf of my employer, IBM, which is a corporate OCA signatory.

**Comment by shruacha1234** [comment]
`/covered`

**Comment by shruacha1234** [comment]
This contribution is on behalf of my employer, IBM, which is a corporate OCA signatory.

File: src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources.java [hunk 1]
```diff
@@ -595,7 +595,7 @@ public class XPATHErrorResources extends ListResourceBundle
   { ER_RTF_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
        "rtf() not supported by XRTreeFragSelectWrapper"},
 
-  { ER_RTF_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
+  { ER_ASNODEITERATOR_NOT_SUPPORTED_XRTREEFRAGSELECTWRAPPER,
```
Comment (JoeWang-Java) [inline src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources.java:598]: Please replace the usage in XRTreeFragSelectWrapper, and also update the LastModified tag.

**Review by JoeWang-Java — APPROVED** [review]
Looks good.

**Comment by shruacha1234** [comment]
`/integrate`

**Comment by backwaterred** [comment]
@shruacha1234 Please don't integrate without addressing the comments left in my review. There are several files that still use the old key incorrectly.

**Comment by backwaterred** [comment]
It also looks like you have not run the pre-tests yet. Please enable them in GitHub Actions and confirm that they pass.

https://wiki.openjdk.java.net/display/SKARA/Testing

**Comment by JoeWang-Java** [comment]
Note that I've run the tests locally. The default test configuration is tier1, which isn't really helpful in this case excepting heating up the planet. If you plan to do more on java.xml, you may run tier2 to verify your changes.

**Review by backwaterred — APPROVED** [review]

**Comment by backwaterred** [comment]
LGTM. Nicely done @shruacha1234

**Comment by JoeWang-Java** [comment]
Changing resource bundles is not required as the L10n resource files update would cover that. As you've modified the files, you'll need to update the license header, using XPATHErrorResources_ja.java as an example and update the year and LastModified tag.

**Comment by JoeWang-Java** [comment]
After the last update, the years in the copyright header and LastModified tag do not match. For example, the header for XPATHErrorResources_de.java starts like: Copyright (c) 2019, 2021, where 2021 needs to be updated to 2022. When I mentioned using XPATHErrorResources_ja.java as an example, I meant replacing "reserved comment block" with the copyright header and then update the copyright year. Sorry if I wasn't clear enough. But basically using that as a template for the format of the header, and keep the year updated to the current. Thanks.

File: src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources_es.java [hunk 2]
```diff
@@ -1,6 +1,5 @@
 /*
- * reserved comment block
- * DO NOT REMOVE OR ALTER!
+ * Copyright (c) 2019, 2022, Oracle and/or its affiliates. All rights reserved.
```
Comment (JoeWang-Java) [inline src/java.xml/share/classes/com/sun/org/apache/xpath/internal/res/XPATHErrorResources_es.java:2]: Remove "2019, ", same with other classes. The header (along with the LastModified tag) serves as an indication that you've modified the file. As this is the first of such modification, it just needs to be noted with the year it's done, that is "2022, ".

**Comment by shruacha1234** [comment]
`/integrate`

**Comment by JoeWang-Java** [comment]
Copyright years for XRTreeFragSelectWrapper.java and XSLTErrorResources.java were still not updated, with the later missing the LastModified tag. Please double check all files before integrate.

**Review by JoeWang-Java — APPROVED** [review]
They all look good now. Thanks.

**Comment by shruacha1234** [comment]
@JoeWang-Java
 Added Copyright header year and Last-Modified tag, with this commit the test is failing could you please suggest the same

**Comment by JoeWang-Java** [comment]
Did you mean the failure (Pre-submit tests - Linux x86 - Test (tier1) ) above?  That's not related to your change.

**Comment by shruacha1234** [comment]
@JoeWang-Java Yes. 
Shall I comment as **integrate** then?

**Comment by JoeWang-Java** [comment]
> @JoeWang-Java Yes. Shall I comment as **integrate** then?

Yes, please go ahead with integrate.

**Comment by shruacha1234** [comment]
`integrate`

**Comment by JoeWang-Java** [comment]
> `integrate`

I think you miss slash, it's slash integrate.

**Comment by JoeWang-Java** [comment]
> `/integrate`

There may be sth. with your input, it's not recognized as Git command. Try again, or copy this command instead.

**Comment by naotoj** [comment]
Do not enclose the skara command within backquotes.

**Comment by JoeWang-Java** [comment]
> Do not enclose the skara command within backquotes.

Thanks Naoto!  I didn't realize that's what he did :-)

**Comment by shruacha1234** [comment]
@JoeWang-Java I have issued the **integrate** command. Now it is ready to be sponsored. Please do the needful

**Pull request closed without merging**